src/scripts/main.js

```diff
@@ -1,3 +1,214 @@
 'use strict';
 
-// write code here
+const thead = document.querySelector('thead');
+const tbody = document.querySelector('tbody');
+
+// table sorting by clicking on the title (in two directions)
+const nodeList = [...tbody.children].map((i) => i.innerText.split('\t'));
+let sortByName = 'asc';
+let sortByPosition = 'asc';
+let sortByOffice = 'asc';
+let sortByAge = 'asc';
+let sortBySalary = 'asc';
+
+thead.onclick = (e) => {
+  const sortBy = e.target.innerText;
+
+  if (sortBy === 'Name') {
```

Comment:
use switch construction here

Reply:
Also, you can join some cases - the sorting logic is the same in a few different columns

Example (schematic):

```
case 'Name':
case 'Position':
  list.sort()
```